app/src/main/java/com/example/myapplication/SecondFragment.java

```diff
@@ -210,6 +212,7 @@ public void onClick(View view) {
             @Override
             public void onClick(View view) {
                 MainActivity.amp ++;
+                UserModel.getMatchData().setAmpTeleop(MainActivity.amp);
```

Comment:
If we are storing ampTeleop in the MatchData class we should remove it from MainActivity. Here it can be something like

    MatchData matchData = UserModel.getMatchData();
    matchData.setAmpTeleop(matchData.getAmpTeleop() + 1);

Reply:
Same comment for other changes in this file